src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,76 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DEFAULT_DELIMETER = ",";
+    private static final String DATA_SUPPLY = "supply";
+    private static final String DATA_BUY = "buy";
+    private static final String DATA_RESULT = "result";
+    private static final int TYPE = 0;
+    private static final int VALUE = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String dataFromFile = readFromFile(fromFileName);
+        String report = processData(dataFromFile);
+        writeToFile(toFileName, report);
+    }
+
+    private String readFromFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+        try (BufferedReader bufferedReader =
+                     new BufferedReader(new FileReader(new File(fromFileName)))) {
```

Comment:
FileReader excepts String, so you dont need `new File()`

Reply:
\+